h/authorization.py

```diff
@@ -32,16 +32,32 @@
 their client credentials, but they do not confer any authorizations and no
 mechanism exists yet for users to grant authorizations to clients.
 """
-from annotator import auth
+from datetime import datetime
+
+import iso8601
+import jwt
+from oauthlib.oauth2 import BearerToken
 from oauthlib.oauth2 import ClientCredentialsGrant
 from oauthlib.oauth2 import RequestValidator as _RequestValidator
 from pyramid.authorization import ACLAuthorizationPolicy
 from pyramid.exceptions import BadCSRFToken
 from pyramid.session import check_csrf_token
+from six import string_types
 
-from .oauth import AnnotatorToken
+from .oauth import JWTBearerGrant
 from .security import WEB_SCOPES
 
+DEFAULT_TTL = 3600
+EPOCH = datetime(1970, 1, 1).replace(tzinfo=iso8601.iso8601.UTC)
+
+
+def now():
```

Comment:
Consider calling this `utc_now()` to reinforce that the timezone is different from just calling `datetime.now()`?